addons/html_editor/static/src/main/link/link_plugin.js

```diff
@@ -206,7 +207,9 @@ export class LinkPlugin extends Plugin {
                         this.overlay.close();
                         this.shared.setSelection(this.shared.getEditableSelection());
                     } else {
+                        const restore = prepareUpdate(...leftPos(this.linkElement));
```

Comment:
Do we also need it for space preservation when pasting a link? like in the function `createLink`

Reply:
I'm not sure... but I don't think so.
Adding stuff next to a visible space will not make it invisible.
But removing stuff next to a visible might make it invisible, so we take care of preserving update.

Reply:
okok, so I will merge it after seb's pr is merged? cause I'm kinda worried that popover check fails again :/